### ISSUE TITLE
Quickly tapping on elements (other than Button) will result in skipped tap events

### ISSUE THREAD

**Issue by tafelnl** [nativescript-vue/nativescript-vue#593]
### Version
2.4.0

### Reproduction link
[https://play.nativescript.org/?template=play-vue&id=4yj9LJ](https://play.nativescript.org/?template=play-vue&id=4yj9LJ)
### Platform and OS info
Android 9.0

### Steps to reproduce
Run the app and click the label saying "tap me". It will output "tapped" into the console.

### What is expected?
When the Label is quickly tapped (twice) it should output "tapped" twice.

### What is actually happening?
It will only output "tapped" once.

---
If this is actually intended behaviour, can someone explain why?

**Comment by rigor789** [comment]
Tapping quickly triggers the `@doubleTap` event, and not the `tap` event. Seems like this is the expected behaviour: https://github.com/NativeScript/NativeScript/issues/7281

**Comment by tafelnl** [comment]
Okay, but why is it triggered twice on iOS then? If it is expected behaviour it should at least be consistent right?

**Comment by tafelnl** [comment]
And the issue you are referring to seems to consider an element with has **both** the @doubleTap and @tap event set.
In this case I am only using the the @tap event

**Comment by rigor789** [comment]
I agree, both platforms should behave the same! However, we're not overriding this behavior in NativeScript-Vue itself, and we should probably move the issue to the NativeScript repo where it needs to be addressed.

**Comment by tafelnl** [comment]
Thanks for answering!
I think you are right then, should move it to NativeScript

**Comment by rigor789** [comment]
If you need a workaround in the meantime, you can use the `touch` event and checking if the `action` is `down`:
https://play.nativescript.org/?template=play-vue&id=4yj9LJ&v=2

**Comment by tafelnl** [comment]
Thanks!
But that is not enough yet, because it already triggers although you did not lift your finger.
So I should probably also check for `@touch` in combination with `action` is `up`

**Comment by rigor789** [comment]
I think just checking for `up` instead of `down` would be enough.

**Comment by tafelnl** [comment]
Oh yeah, that's actually quite smart, keep it stupid simple they say  :)
